lib/msal-node/src/retry/LinearRetryPolicy.ts

```diff
@@ -39,4 +40,29 @@ export class LinearRetryPolicy implements IHttpRetryPolicy {
 
         return false;
     }
+
+    retryAfterMillisecondsToSleep(
```

Comment:
nit: Please add explicit visibility modifiers (public, private) to help reviewers who are not familiar with TS review.

Reply:
The visibility of all properties or methods in Typescript classes is “public“ by default